Apps/Database/src/Context/GatewayDbContext.cs

```diff
@@ -261,6 +262,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                     v => EnumUtility.ToEnumString<LegalAgreementType>(v, true),
                     v => EnumUtility.ToEnum<LegalAgreementType>(v, true)));
 
+            modelBuilder.Entity<Dependent>()
+                .HasKey(dependent => new { dependent.ParentHdId, dependent.DependentHdId });
```

Comment:
No, there could be multiple instances of the dependent Id (for example both parents add their children)

Reply:
Not sure as I may have missed this. It is probably better to check with @ninosamson for this scenario. If we allows this scenario, I would prefer to add 2 foreign keys max from Dependent table to User table to avoid dup. data.